Api/V8/Config/Services/controllers.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Api\V8\Controller;
+use Interop\Container\ContainerInterface as Container;
+
+return [
+    Controller\UtilityController::class => function (Container $container) {
+        return new Controller\UtilityController(
+            $container->get(\League\OAuth2\Server\AuthorizationServer::class)
+        );
+    },
+    AuthenticationController::class => function () {
+        require_once 'modules/Users/authentication/AuthenticationController.php';
```

Comment:
use __DIR__ constant to make sense the full path

Reply:
Fixed